vuecomponents/bundle/src/main/bundle/dynamic-criterion_eu.properties

```diff
@@ -0,0 +1,8 @@
+select_criterion=Haga clic aquí para marcar el criterio como seleccionado para el estudiante
+criterion_points=Puntos del criterio
+criterion_description=Descripción del criterio
+confirm_criterion=Confirmar cambios para este criterio
+remove_criterion=Borrar este criterio
+edit_criterion=Editar este criterio
+confirm_remove=¿Estás seguro de querer borrar el criterio? Es posible que se haya seleccionado para otros estudiantes, que verán su nota modificada al confirmar los cambios en los criterios.
+criterion_zero=No se puede tener un criterio con valor 0
```

Comment:
Hi @PabloSanRoman, could you help with these translations please?

Reply:
+select_criterion=Klik egin hemen irizpidea markatuta uzteko ikaslearentzat
+criterion_points=Irizpidearen puntuak
+criterion_description=Irizpidiearen deskribapena
+confirm_criterion=Berretsi irizpide honen aldaketak
+remove_criterion=Ezabatu irizpidea
+edit_criterion=Editatu irizpidea
+confirm_remove=Ziur zaude ezabatu nahi duzula irizpidea? \
Berresten baduzu ikasle batzuen notak aldatuko dira baldin eta irizpide hau hautatua duten. 
+criterion_zero=0 balorea duen irizpidea ezinezkoa da.